ProcessMaker/processmaker
# Fix CSS Parsing Issue When Applying Screen Templates with Inline Comments in Screen Builder

## Issue & Reproduction Steps
This PR resolves an issue where CSS comments from screen templates were causing improper CSS formatting when applied to an existing screen. Specifically, the closing comment tag (*/) was removed, which resulted in unintended commenting of CSS code. The fix refines the parsing logic to correctly handle CSS inline comments during template application.

## Solution
- Refactor CSS parsing logic and improve the regex to properly handle and retain inline comments when applying screen template CSS to an existing screen.

## How to Test

1. Run the command `php artisan processmaker:sync-screen-templates` to sync available screen templates.
2. Create a new blank screen and open screen builder.
3. In the Screen Builder, select the 'Templates' option in the top navigation menu.
4. Locate and select the 'Paper Template' screen template (known for causing comment-related issues).
5. Apply the CSS from the selected template to the blank screen.
6. After applying the CSS, select the 'CSS' option in the top navigation menu.
7. Ensure that the CSS is properly formatted, especially checking that comments are intact and the closing comment tags (*/) are correctly present.

## Related Tickets & Packages
[- FOUR-19270](https://processmaker.atlassian.net/browse/FOUR-19270)

## Code Review Checklist
